foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/ConfigStoreInit.java

```diff
@@ -0,0 +1,57 @@
+package com.boozallen.aissemble.configuration;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Configuration::Store
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.configuration.store.ConfigLoader;
+import com.boozallen.aissemble.configuration.store.Property;
+import io.quarkus.runtime.Startup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.spi.CDI;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * ConfigStoreInit loads configuration properties as application starts up
+ */
+@Startup
+@ApplicationScoped
+public class ConfigStoreInit {
+    private static final Logger logger = LoggerFactory.getLogger(ConfigStoreInit.class);
+
+    @PostConstruct
+    void init() {
+        logger.info("Initialize store configuration properties...");
+        String baseUri = System.getenv("BASE_URI");
+        String environmentUri = System.getenv("ENVIRONMENT_URI");
+        try {
+            ConfigLoader configLoader = CDI.current().select(ConfigLoader.class,new Any.Literal()).get();
+            Set<Property> properties = null;
+            if (environmentUri != null && !environmentUri.isEmpty()) {
+                properties = configLoader.loadConfigs(baseUri,environmentUri);
+            } else if (baseUri != null && !baseUri.isEmpty()){
+                properties = configLoader.loadConfigs(baseUri);
+            } else {
+                logger.warn("No properties are loaded. To load properties, you need to set the environment variable BASE_URI and ENVIRONMENT_URI");
```

Comment:
S: I might actually throw an exception here.  Unless I'm missing something, isn't the whole point of this service to load these in, so if it can't do that it's failing at its primary function?